SolrNet.Tests/LukeResponseParserTest.cs

```diff
@@ -0,0 +1,50 @@
+using System.Xml.Linq;
+using MbUnit.Framework;
+using SolrNet.Schema;
+using SolrNet.Tests.Utils;
+
+namespace SolrNet.Tests 
+{
+    [TestFixture]
```

Comment:
We are now using XUnit  instead of Visual Studio unit tests, could you move this over?